server/src/Rotation.ts

```diff
@@ -0,0 +1,11 @@
+export default class Rotation {
+    roll: number;
+    pitch: number;
+    yaw: number;
+
+    constructor(roll: number = 0, pitch: number = 0, yaw: number = 0) {
```

Comment:
same as in `Position`, set the constructor params as public, and you don't need this.

Reply:
Done.